File: 342. Power of Four.js
/**
 * @param {number} num
 * @return {boolean}
 */
var isPowerOfFour = function(num) {
    if (num <= 0) return false;
    var temp = Math.log10(num) / Math.log10(4);
    if (Math.round(temp) == temp) return true;
    return false;
};

File: 342. Power of Four.py
import math
class Solution(object):
    def isPowerOfFour(self, num):
        """
        :type num: int
        :rtype: bool
        """
        if num <= 0:
            return False
        temp = math.log10(num) / float(math.log10(4))
        return True if int(temp) == temp else False
